app/modules/enduser_setup.c

```diff
@@ -362,8 +376,15 @@ static void enduser_setup_check_station(void *p)
     return;
   }
 
-  enduser_setup_connected_callback(lua_getstate());
-  enduser_setup_stop(NULL);
+  enduser_setup_connected_callback();
+
+  /* Trigger shutdown, but allow time for HTTP client to fetch last status. */
+  if (!manual)
+  {
+    enduser_setup_check_station_stop();
```

Comment:
`enduser_setup_check_station_stop()` needs to be outside of the `if` block so that the callback isn't executed every second while in manual mode. ;-)